articles/extensions/delegated-admin/hooks.md

```diff
@@ -299,6 +316,147 @@ The **Settings Query** allows you to customize the look and feel of the extensio
 
 Example:
 
+```js
+function(ctx, callback) {
+  var department = ctx.request.user.app_metadata && ctx.request.user.app_metadata.department;
+
+  return callback(null, {
+    // Only these connections should be visible in the connections picker.
+    // If only one connection is available, the connections picker will not be shown in the UI.
+    connections: [ 'Username-Password-Authentication', 'My-Custom-DB' ],
+    // The dictionary allows you to overwrite the title of the dashboard and the "Memberships" label in the Create User dialog.
+    dict: {
+      title: department ? department + ' User Management' : 'User Management Dashboard',
+      memberships: 'Departments',
+      menuName: ctx.request.user.name
+    },
+    // The CSS option allows you to inject a custom CSS file depending on the context of the current user (eg: a different CSS for every customer)
+    css: (department && department !== 'IT') && 'https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css'
+  });
+}
+```
+
+## Custom Fields
+
+Starting with version 3.0 of the Delegated Admin Extension you can now specify custom fields.  These custom fields can be stored anywhere in user_metadata or app_metadata when creating,
+updating, or displaying users.  You can use these fields to search for users and display them in the search results.  Also, it allows you to customize existing fields, such as email, username,
+name, and connection.
+
```

Comment:
Beginning with version 3.0 of the Delegated Admin Extension, you can use custom fields when creating, updating, or displaying information about users.

Auth0 stores the information in custom fields in **user_metadata** or **app_metadata**. These fields are searchable.

Custom fields also allow you to modify existing fields, such as email, username, name, or connection.